### PR TITLE
Cater for home.parity hostname in dappsUrl

### PR DESCRIPTION
**Pull request by jacogr**
Closes https://github.com/ethcore/parity/issues/3337

### CODE REVIEW

**Comment by jacogr** [comment]
@tomusdrw Can you please take a look.

File: js/src/secureApi.js [hunk 1]
```diff
@@ -26,6 +26,7 @@ export default class SecureApi extends Api {
     this._connectState = sysuiToken === 'initial' ? 1 : 0;
     this._needsToken = false;
     this._dappsPort = 8080;
+    this._dappsInterface = null;
```
Comment (tomusdrw) [inline js/src/secureApi.js]: Could default to `window.location.hostname`

Comment (jacogr) [inline js/src/secureApi.js]: Agreed. Had it that way around as a first stab. Just wanted to have additional protection against the case if we ever get null back for whatever reason. (Which would point to more serious issues, but ...)

**Comment by jacogr** [comment]
Merged in master for test failures.

**Pull request merged**